src/theme/colors.ts

```diff
@@ -47,4 +47,16 @@ export const colors = {
     red: '#F7414126',
     yellow: '#FFF72D26',
   },
+
```

Comment:
I'm not exactly sure if breaking out the colors by component is a good idea. But this is how I did this one. Any feedback welcomed.

Reply:
I don't mind this at all. In fact, this may be *the* way to organise our colours so it's more obvious how they're being used in the app's theme. And I don't mind if this leads to a few duplicate colour strings either.